packages/casl-ability/src/builder.js

```diff
@@ -40,20 +51,25 @@ export class AbilityBuilder {
     };
   }
 
-  constructor() {
+  constructor({ subjectName = getSubjectName } = {}) {
     this.rules = [];
+    this[PRIVATE_FIELD] = {
```

Comment:
`AbilityBuilder` is not designed to be instantiated with `new`, so users never should have access to its internals. So, hiding implementation under symbol is not required. Lets just assign `subjectName` directly to `this`